src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;

```

Comment:
Czy jest potrzebny flexbox dla tej kartki ? Przecież, masz dość prosty flow.
Moim zdaniem, lepiej skorzystać z zwykłego modelu blokowego.

Reply:
Użyłem flexboxa głównie po to, aby wyśrodkować elementy, tak jak np. zdjęcie